src/rai/rai/tools/debugging_assistant.py

```diff
@@ -0,0 +1,81 @@
+# Copyright (C) 2024 Robotec.AI
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#         http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import streamlit as st
+from langchain_core.messages import AIMessage, HumanMessage, ToolMessage
+
+from rai.agents.conversational_agent import create_conversational_agent
+from rai.agents.integrations.streamlit import get_streamlit_cb, streamlit_invoke
+from rai.tools.ros.debugging import (
+    ros2_action,
+    ros2_interface,
+    ros2_node,
+    ros2_service,
+    ros2_topic,
+)
+from rai.utils.model_initialization import get_llm_model
+
+
+def initialize_graph():
+    llm = get_llm_model(model_type="complex_model", streaming=True)
+    agent = create_conversational_agent(
+        llm,
+        [ros2_topic, ros2_interface, ros2_node, ros2_service, ros2_action],
```

Comment:
How about other commands? Do you plan to support it? 
It would be good to address this in the docs.
```bash
robo-pc-005 ➜  12_rai git:(feat/debugging-tools) ✗ ros2 --help                              
usage: ros2 [-h] [--use-python-default-buffering] Call `ros2 <command> -h` for more detailed usage. ...

ros2 is an extensible command-line tool for ROS 2.

options:
  -h, --help            show this help message and exit
  --use-python-default-buffering
                        Do not force line buffering in stdout and instead use the python default buffering, which might be affected by PYTHONUNBUFFERED/-u and depends on whatever stdout is interactive or not

Commands:
  action     Various action related sub-commands
  bag        Various rosbag related sub-commands
  component  Various component related sub-commands
  daemon     Various daemon related sub-commands
  doctor     Check ROS setup and other potential issues
  interface  Show information about ROS interfaces
  launch     Run a launch file
  lifecycle  Various lifecycle related sub-commands
  multicast  Various multicast related sub-commands
  node       Various node related sub-commands
  param      Various param related sub-commands
  pkg        Various package related sub-commands
  run        Run a package specific executable
  security   Various security related sub-commands
  service    Various service related sub-commands
  topic      Various topic related sub-commands
  wtf        Use `wtf` as alias to `doctor`
```

Reply:
I wasn't planning on adding anything right now, please specify which tools you would like to see. I will add them when I am free (most likely not this week).

Reply:
I think current coverage is fine for now